sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueMetricsProvider.cs

```diff
@@ -52,6 +52,11 @@ public async Task<int> GetQueueLengthAsync()
                     _logger.LogWarning($"Error querying for queue scale status: {ex.Message}");
                 }
             }
+            catch (Exception ex)
+            {
+                _logger.LogWarning($"Fatal error querying for queue scale status: {ex.Message}");
```

Comment:
I know the existing code was already following this pattern, but it looks like we are throwing away the stack by logging only the exception message. Does this component support logging the full exception object  - with perhaps the StackTrace landing in an appropriate column in the scale controller kusto table?

Reply:
Hmm good point but I don't believe it does currently -- we can also log `Exception.ToString()` instead, which should capture the entire exception trace.

Reply:
Updated to use ToString()